client/reader/following/controller.js

```diff
@@ -1,45 +1,7 @@
-import { isEnabled } from '@automattic/calypso-config';
 import page from '@automattic/calypso-router';
 import { getQueryString } from '@wordpress/url';
-import i18n from 'i18n-calypso';
-import AsyncLoad from 'calypso/components/async-load';
-import { sectionify } from 'calypso/lib/route';
-import { trackPageLoad, setPageTitle } from 'calypso/reader/controller-helper';
 
-const analyticsPageTitle = 'Reader';
-
-const exported = {
-	followingManage( context, next ) {
-		if ( isEnabled( 'subscription-management-redirect-following' ) ) {
-			const queryString = getQueryString( window.location.href );
-			return page.redirect( '/read/subscriptions' + ( queryString ? '?' + queryString : '' ) );
-		}
-
-		const basePath = sectionify( context.path );
-		const fullAnalyticsPageTitle = analyticsPageTitle + ' > Manage Followed Sites';
-		const mcKey = 'following_manage';
-		const { q: sitesQuery, s: subsQuery, sort: subsSortOrder, showMoreResults } = context.query;
-
-		setPageTitle( context, i18n.translate( 'Manage followed sites' ) );
-
-		trackPageLoad( basePath, fullAnalyticsPageTitle, mcKey );
-
-		context.primary = (
-			<AsyncLoad
-				require="calypso/reader/following-manage"
```

Comment:
This `FollowingManage` component can now be removed - this is its last usage. It's a relatively large component - perhaps we can remove some additional unused code with its removal?

Reply:
I know, it will be a nice follow-up 🙂 I didn't do it right away because there is another component that imports from `following-manage/search-followed`. And I didn't want to do a change that not as simple as just deleting the entire folder.

Reply:
Cleaned it up in https://github.com/Automattic/wp-calypso/pull/86996